src/utilities/export/export.ts

```diff
@@ -10,26 +10,26 @@ export function getUnixTimestamp() {
 }
 
 export function exportTsvFile(imageName: string, plotName: string, content: string) {
-    const tsvData = `data:text/tab-separated-values;charset=utf-8,${content}\n`;
+    const tsvData = `data:text/tab-separated-values;charset=utf-8,${content}\n`.trim();
     const dataURL = encodeURI(tsvData).replace(/#/g, "%23");
 
     const a = document.createElement("a") as HTMLAnchorElement;
     a.href = dataURL;
 
-    a.download = `${imageName}-${plotName.replace(" ", "-")}-${getTimestamp()}.tsv`;
+    a.download = `${imageName.replace(" ", "__")}-${plotName.replace(" ", "-")}-${getTimestamp()}.tsv`;
```

Comment:
This will only replace the first space. Perhaps use `replaceAll`. Same for the other function.